google-cloud-asset/samples/acceptance/quickstart_test.rb

```diff
@@ -154,14 +154,13 @@
       object_name = "ruby-analysis-samples.json"
       uri = "gs://#{bucket.name}/#{object_name}"
       assert_nil bucket.file(uri)
-      out, _err = capture_io do
+      _out, _err = capture_io do
```

Comment:
Please retain the assertions, both for this test and the one below. (I assume the actual output is the same, but we just don't want to reference the metadata in the sample.)